addons/packages/antrea/1.7.1/bundle/config/kapp-config.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: kapp.k14s.io/v1alpha1
+kind: Config
+rebaseRules:
+- path: [data]
+  type: copy
+  sources: [existing, new]
+  resourceMatchers:
+  - kindNamespaceNameMatcher: {kind: ConfigMap, namespace: kube-system, name: antrea-ca}
+
+- path: [spec, caBundle]
+  type: copy
+  sources: [existing, new]
+  resourceMatchers:
+  - anyMatcher:
+      matchers:
+      - kindNamespaceNameMatcher: {kind: APIService, namespace: kube-system, name: v1alpha1.stats.antrea.io}
```

Comment:
please review these apiservices to see if there are some deprecated or not

Reply:
https://github.com/antrea-io/antrea/blob/v1.7.1/docs/api.md According to this, we have the correct list.